packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -366,6 +390,19 @@ export const useTransactionHistory = (address: `0x${string}` | undefined) => {
     }
   )
 
+  const transactions: MergedTransaction[] = useMemo(() => {
```

Comment:
In the future, we might need to replace SWR? It's not the first time we need to manually set the cache, we might want to look at React Query if it handles it better